crates/polars-plan/src/plans/conversion/ir_to_dsl.rs

```diff
@@ -452,9 +452,9 @@ pub fn ir_function_to_dsl(input: Vec<Expr>, function: IRFunctionExpr) -> Expr {
                 #[cfg(feature = "string_reverse")]
                 IB::Reverse => B::Reverse,
                 #[cfg(feature = "string_pad")]
-                IB::PadStart { length, fill_char } => B::PadStart { length, fill_char },
```

Comment:
Could you add a `is_scalar_ae(e[1])` check here as well? That will cause it to error when planning the queries and not when executing the query.

Reply:
Oh nevermind, they are not necessarily scalars. :smile: